src/code-block.js

```diff
@@ -13,14 +14,13 @@ const preload = (content, resourcePath) => {
     return content;
   }
 
-  codeBlocks.each((i, elem) => {
+  codeBlocks.each((idx, elem) => {
     const $elem = $(elem);
-    const innerText = $elem.html()
-      .toString()
+      let content = jsonDataUtil.parseAllBuildTimeBindings($elem.html())
       .replace(/{/g, `{{ '{' }}`)
       .replace(/</g, '&lt;');
 
-    $elem.text(innerText);
+    $elem.html(content);
```

Comment:
Somehow the `.text()` was throwing an invalid string length error.   As we are parsing it as HTML to begin with, I figured it made sense to set it back to html, since we are returning the whole thing `.toString();` after this step anyway.